components/shared/embedder/input_events.rs

```diff
@@ -280,3 +284,62 @@ pub enum GamepadUpdateType {
     /// <https://www.w3.org/TR/gamepad/#dfn-represents-a-standard-gamepad-button>
     Button(usize, f64),
 }
+
+pub trait WebDriverInputEvent {
+    fn webdriver_msg_id(&self) -> Option<WebDriverMessageId>;
+    fn with_webdriver_msg_id(&mut self, webdriver_id: Option<WebDriverMessageId>) -> Self;
+}
```

Comment:
You do not need to add a trait to do this. You can simply do `impl InputEvent`, but please please add these methods to the existing `impl InputEvent` block. Thanks!